io/disk/mvcli_test.py

```diff
@@ -0,0 +1,277 @@
+                vd_list.append(obj.group(1))
+        if vd_list is None:
+            self.log.info("Not able to found any VD's are found")
+        self.log.info("List of VD's to be deleted: %s", vd_list)
+        for vd in vd_list:
+            self.run_command("%s delete -o vd -i %s" % (self.base_force, vd))
+        output = self.run_cmd_output("%s info -o vd" % self.base)
+        if "No virtual disk is found." in output:
+            self.log.info("Mvcli delete operation success")
+        else:
+            self.fail("Mvcli delete operation failed")
+
+    def test_create(self):
+        """
+        Create virtual disk on supported raid levels
+        and try to delete VD which was created
+        for RAID 0, Use all disks from PD,
+        for RAID 1, Use minimum of two disks from PD
+        for RAID 10, Use minimum of four disks from PD
+        """
+        raid_levels = ["0", "1", "10"]
+        base = "%s create -o vd " % self.base_force
+        pd_list_len = len(self.pd_ids)
+        self.log.info("PD List %s, length: %s", self.pd_ids, pd_list_len)
+        for level in raid_levels:
+            name = "MyRaid_%s" % level
+            if level == "0":
+                pd_list = ",".join(self.pd_ids)
+            elif level == "1":
+                if pd_list_len < 2:
+                    continue
+                pd_list = "%s,%s" % (self.pd_ids[0], self.pd_ids[1])
+            elif level == "10":
+                if pd_list_len < 4:
+                    continue
+                pd_list = "%s,%s,%s,%s" % (self.pd_ids[0], self.pd_ids[1],
+                                           self.pd_ids[2], self.pd_ids[3])
+            cmd = "%s -r%s -d %s -n %s" % (base, level, pd_list, name)
+            self.log.info("Mvcli RAID create level: %s, pd_list: %s, name: %s",
+                          level, pd_list, name)
+            self.log.info("Mvcli RAID VD create: %s", cmd)
+            self.run_command(cmd)
+            self.log.info("Mvcli RAID VD create success")
+            self.run_command("%s info -o vd" % self.base)
+            self.run_command("%s get -o vd -i 0" % self.base)
+            self.run_command("%s delete -o vd -i 0" % self.base_force)
+            self.log.info("Mvcli RAID VD delete success")
+            output = self.run_cmd_output("%s info -o vd" % self.base)
+            if "No virtual disk is found." in output:
+                self.log.info("Mvcli delete operation success")
+            else:
+                self.fail("Mvcli delete operation failed")
+
+    def test_event(self):
+        """
+        Get the current events.
+        """
+        self.log.info("Mvcli event test")
+        self.run_command("%s event -s 10" % self.base)
+
+    def tearDown(self):
+        """
+        Gather any kernels errors at the end of test
+        """
+        self.log.info("Gathering any kernel errors")
+        self.run_command("dmesg -T --level=alert,crit,err,warn")
+        self.test_delete()
```

Comment:
@pridhiviraj small change. `test_delete()` should be called before collecting dmesg logs